frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -509,3 +509,14 @@ button.usa-pagination__button.usa-button {
 .usa-accordion__heading {
   border-bottom: 2px solid color(gray-warm-30);
 }
+
+#search-filter-drawer {
+  text-align: right;
```

Comment:
Could these all be added as individual classes to stay consistent with how we use the uswds classes?